packages/salesforcedx-utils-vscode/src/commands/commandletExecutors.ts

```diff
@@ -161,6 +162,9 @@ export abstract class LibraryCommandletExecutor<T>
     const startTime = process.hrtime();
     const channelService = new ChannelService(this.outputChannel);
     const telemetryService = TelemetryService.getInstance();
+    if (SfdxSettingsService.getEnableClearOutputBeforeEachCommand()) {
```

Comment:
The LibraryCommandletExecutor is exported from `salesforcedx-utils-vscode` and is another place that I found to check and clear the output log.